Source/dead.cpp

```diff
@@ -76,7 +76,7 @@ void AddDead(Point tilePosition, int8_t dv, Direction ddir)
 	dDead[tilePosition.x][tilePosition.y] = (dv & 0x1F) + (ddir << 5);
 }
 
-void SetDead()
+void SyncUniqDead()

```

Comment:
Can we avoid abbreviations? Is this for unique monsters? If so, can we use "Unique"?